### ISSUE TITLE
UnicodeEncodeError on getTransactionDetailsRequest

### ISSUE THREAD

**Issue by rneilson** [AuthorizeNet/sdk-python#94]
Possibly related to #73, but at a different point in `execute()`.

For transactions with non-ASCII characters, I'm getting `UnicodeEncodeError: 'ascii' codec can't encode characters in position 873-874: ordinal not in range(128)` when retrieving transaction details.

```
  File "[virtualenv]/lib/python2.7/site-packages/authorizenet/apicontrollersbase.py", line 156, in execute
    domResponse = xml.dom.minidom.parseString(self._httpResponse)
  File "/usr/lib64/python2.7/xml/dom/minidom.py", line 1928, in parseString
    return expatbuilder.parseString(string)
  File "/usr/lib64/python2.7/xml/dom/expatbuilder.py", line 940, in parseString
    return builder.parseString(string)
  File "/usr/lib64/python2.7/xml/dom/expatbuilder.py", line 223, in parseString
    parser.Parse(string, True)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 876-877: ordinal not in range(128)
```

#75 added `xml.dom.minidom.parseString(self._httpResponse.encode('utf-8').decode('utf-8'))` when logging an error [see here](https://github.com/AuthorizeNet/sdk-python/blob/master/authorizenet/apicontrollersbase.py#L162), but the [initial response parsing](https://github.com/AuthorizeNet/sdk-python/blob/master/authorizenet/apicontrollersbase.py#L156) doesn't do so, and raises an uncaught exception.

**Comment by Sapbasu15** [comment]
@rneilson please refer to the pull request #78 merged for this fix. The initial response parsing was not required and hence removed. Also decode is not required as parseString expects UTF-8 encoded string this case. This will be out in the next python sdk release.

**Comment by rneilson** [comment]
@Sapbasu15 Thanks. Any word on when said release will be? Currently having to do some ugly workarounds for this...

**Comment by Sapbasu15** [comment]
@rneilson  We'll try to do a release soon, but no exact date decided yet. There are some other change requests also which we are trying to accomodate in the next release.

**Comment by Sapbasu15** [comment]
Fix for the issues is released in PR #102 and has been released in the latest version.
